### PR TITLE
Automatically port System.Text.Json.JsonValueKind

### PR DESCRIPTION
**Pull request by carlossanlop**
New batch.

### CODE REVIEW

File: xml/System.Text.Json/JsonValueKind.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@
       </ReturnValue>
       <MemberValue>7</MemberValue>
       <Docs>
-        <summary>Indicates that a value is the JSON value <see langword="null" />.</summary>
+        <summary>The JSON value <see langword="null" />.</summary>
```
Comment (ahsonkhan) [inline xml/System.Text.Json/JsonValueKind.xml]: These shouldn't be langwords (like discussed previously). Same with true/false. They are JSON-specific literals. I think we decided on marking them bold instead (if I recall correctly).

File: xml/System.Text.Json/JsonValueKind.xml [hunk 2]
```diff
@@ -54,7 +54,7 @@
       </ReturnValue>
       <MemberValue>6</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Indicates that a value is the JSON value <see langword="false" />.</summary>
```
Comment (ahsonkhan) [inline xml/System.Text.Json/JsonValueKind.xml]: @rpetrusha - looks like you missed updating "False" in your effort to "shorten the text".

**Comment by ahsonkhan** [comment]
Do you know when the changes go live?
The PR link leads to 404 and I can't see the update on the main docs site:
https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonvaluekind?branch=pr-en-us-2877

https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonvaluekind?view=netcore-3.0
image

**Comment by mairaw** [comment]
We push the daily changes live at the end of the day. First this one needs to finish building https://github.com/dotnet/samples/pull/1134 and then I'll start the other repos.

**Comment by mairaw** [comment]
And once it's merged to master, you need to change the query param to master to see it:
https://review.docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonvaluekind?branch=master&view=netcore-3.0

**Pull request merged**